platform/security/src/main/java/org/codice/alliance/security/token/videographer/VideographerAuthenticationToken.java

```diff
@@ -0,0 +1,54 @@
+package org.codice.alliance.security.token.videographer;
+
+import org.apache.commons.lang3.StringUtils;
+import org.codice.alliance.security.principal.videographer.VideographerPrincipal;
+import org.codice.ddf.security.handler.api.BSTAuthenticationToken;
+
+import ddf.security.common.audit.SecurityLogger;
+
+public class VideographerAuthenticationToken extends BSTAuthenticationToken {
+
+    public static final String VIDEOGRAPHER_CREDENTIALS = "Videographer";
+
+    public static final String BST_VIDEOGRAPHER_LN = "Videographer";
```

Comment:
This was how the original GuestAuthenticationToken code was written. I just copied it and changed the values.